README.md

```diff
@@ -8,13 +8,32 @@ ContribKanban is an application which mirrors Drupal.org issues into kanban boar
 
 Want to contrib to ContribKanban? Local development is powered by [Lando](https://lando.dev/).
 
+### lando
+
 ```shell
 lando start
 ```
 
+### ddev
+
+1. Configure frontend app.
+    - Create `.env.local` in frontend directory
+    - Add following environment variables
+
+      ```
+      REACT_APP_API_BASE_URL=https://contribkanban.com.ddev.site
+      PORT=4000
+      ```
+
```

Comment:
We can add `REACT_APP_API_BASE_URL` to the DDEV config yaml web environment variables